plugins/outputs/groundwork/groundwork.go

```diff
@@ -222,6 +227,7 @@ func (g *Groundwork) parseMetric(metric telegraf.Metric) (string, *transit.Dynam
 		},
 		Status:           transit.MonitorStatus(status),
 		LastCheckTime:    lastCheckTime,
+		NextCheckTime:    nextCheckTime,
```

Comment:
Would this also be accepted by the server?
```suggestion
		NextCheckTime:    lastCheckTime,
```

Reply:
Yes, but we have such a field on the UI side, which can be confusing as it coincides with the LastCheckTime.

Reply:
Then at least it is visible to not count too much on it. Now it is just making up values..

Reply:
Ok, let's leave it the same as LastCheckTime.